docs/webapi/waitForNotAttached.mustache

```diff
@@ -0,0 +1,9 @@
+Waits for an element to become not attached to the DOM on a page (by default waits for 1sec).
+Element can be located by CSS or XPath.
+
+```
+I.waitForNotAttached('#popup');
```

Comment:
Can you provide a better naming? By `attached` I'd think about attached file. And yes, we have `attachFile` method so this can be confusing. How about `waitForDetached` ?